djangocms_versioning/admin.py

```diff
@@ -355,6 +355,7 @@ def queryset(self, request, queryset):
 class VersionAdmin(admin.ModelAdmin):
     """Admin class used for version models.
     """
+    change_list_template = "admin/djangocms_versioning/change_list.html"
```

Comment:
I don't think that we need this. The changes should be added to the [VersionAdminMixin](https://github.com/django-cms/djangocms-versioning/blob/13bfead550c71b78a64c36500039f010f799bcd9/djangocms_versioning/admin.py#L117) , This Mixin is reused by the other 3rd party packages to add the admin actions to the packages. 

The package djangocms-page-admin implementation for the burger cam be left as it is because it is an older implementation pre the mixin which is probably what's confused you.